Source/WTF/wtf/ListHashSet.h

```diff
@@ -883,6 +887,20 @@ inline auto ListHashSet<T, U>::makeConstIterator(Node* position) const -> const_
     return const_iterator(this, position);
 }
 
+template<typename T, typename U>
+template<typename OtherCollection>
+inline bool ListHashSet<T, U>::operator==(const OtherCollection& otherCollection) const
```

Comment:
I find this operator very confusing, what will it be used for? The point of a ListHashSet (vs a HashSet) is that it preserves ordering. However, this new operator doesn't care about ordering.

Reply:
I don't know which behavior would be better but it is confusing, one wouldn't obviously know what to expect here.